src/main/resources/add-ons-to-index.json

```diff
@@ -1558,6 +1558,39 @@
         "package": "facelist"
       }
     },
+    {
+      "uid":"org.openmrs.module.spa",
+      "type":"OMOD",
+      "name":"Single Page Application",
+      "maintainers":[
+        {
+          "name":"Daniel Kayiwa"
+        },
+        {
+          "name":"Burke Mamlin"
+        },
+        {
+          "name":"Brandon Istenes"
+        }
+      ],
+      "links":[
+        {
+          "rel":"documentation",
+          "href":"https://wiki.openmrs.org/display/projects/Frontend+Implementer+Documentation",
```

Comment:
Is this the project home page?

Reply:
This project does not have specific homepage, currently it is using the same homepage Microfrontends